lib/catalog/api/minion/logging.rb

```diff
@@ -8,7 +8,12 @@ class << self
       end
 
       def self.logger
-        @logger ||= ManageIQ::Loggers::CloudWatch.new
+        if ENV["LOG_TO_STDOUT"].present?
```

Comment:
@syncrou How would we set this LOG_TO_STDOUT environment variable, is it already being set in the deployment config for the pod?

Reply:
We can manually edit the deployment environment variables in the Openshift UI. That will force a new deployment but keep the manual changes.

The plan would be to add this env variable to the e2edeploy script for CI and for the other environments turn on the environment variable manually if / when it would be needed.